packages/messaging/test/controller-delete-token.test.ts

```diff
@@ -205,7 +197,7 @@ describe('Firebase Messaging > *Controller.deleteToken()', function() {
     it(`should handle error on deleteToken ${ServiceClass.name}`, function() {
       const fakeSubscription = {
         endpoint: EXAMPLE_TOKEN_SAVE.endpoint,
-        unsubscribe: () => Promise.resolve()
+        unsubscribe: async () => {}
```

Comment:
I'm personally not a fan of this pattern - it's difficult to tell what the stubbed behavior should be doing.

Reply:
It clearly states "do nothing", doesn't it?

Reply:
It's more about the expectation of the API (i.e. return a Promise).